src/components/molecule/ToDoItem/ToDoItem.tsx

```diff
@@ -4,12 +4,12 @@ import KebabMenu from '../../atom/KebabMenu'
 import { ToDoItemContainer, ToDoItemText } from './elements'
 import { ToDoItemProps } from './types'
 
-const ToDoItem = ({onDelete,onUpdate,value,onSelect,selected,selectionMode,completed,deleteDrawer}: ToDoItemProps) => {
+const ToDoItem = ({onDelete,onUpdate,value,onSelect,selected,selectionMode,completed,deleteDrawer,onTextClick}: ToDoItemProps) => {
         const [popupmenu,setpopupmenu] = useState(false);
   return (
     <ToDoItemContainer className={selectionMode?" ":"justified"}>
         {completed?" ":selectionMode?<CheckBox checked={selected||false} onChange={onSelect}/>:" "}
-        <ToDoItemText className={completed?"completed":popupmenu?"popupmenu":deleteDrawer?"deletedrawer":" "}>{value}</ToDoItemText>
+        <ToDoItemText className={completed?"completed":popupmenu?"popupmenu":deleteDrawer?"deletedrawer":" "} onClick={selectionMode?()=>{}:onTextClick}>{value}</ToDoItemText>
```

Comment:
I think this would be more readable if you create create a separate function before `return` like this
```
const onClick = () => {
   if(selectionMode) onTextClick();
}
...
<ToDoItemText onClick={onClick} ...>
...
```